test/simple-concat-test.js

```diff
@@ -0,0 +1,143 @@
+var concat = require('..');
+var fs = require('fs-extra');
+var path = require('path');
+var broccoli = require('broccoli');
+var expectFile = require('./helpers/expect-file');
+
+var chai = require('chai');
+var chaiFiles = require('chai-files');
+var chaiAsPromised = require('chai-as-promised');
+
+chai.use(chaiFiles);
+chai.use(chaiAsPromised);
+
+var expect = chai.expect;
+var file = chaiFiles.file;
+
+var firstFixture = path.join(__dirname, 'fixtures', 'first');
+var emptyFixture = path.join(__dirname, 'fixtures', 'empty');
+
+describe('simple-concat', function() {
+  var builder;
+
+  afterEach(function() {
+    if (builder) {
+      return builder.cleanup();
+    }
+  });
+
+  it('inserts header when sourcemaps are disabled', function() {
+    var node = concat(firstFixture, {
+      header: "/* This is my header. */",
+      inputFiles: ['**/*.js'],
+      outputFile: '/all-with-header.js',
+      sourceMapConfig: { enabled: false }
+    });
+    builder = new broccoli.Builder(node);
+    return builder.build().then(function(result) {
+      expectFile('all-with-header.js').withoutSrcURL().in(result);
+      expectFile('all-with-header.map').notIn(result);
+    });
+  });
+
+  it('disables sourcemaps when requested', function() {
```

Comment:
test name seems somewhat odd considering all tests in this file are non-sourcemap